cmd/image.go

```diff
@@ -92,6 +101,11 @@ Example usage:
 kubeaudit image --image gcr.io/google_containers/echoserver:1.7
 kubeaudit image -i gcr.io/google_containers/echoserver:1.7`,
 	Run: func(cmd *cobra.Command, args []string) {
+		if len(imgConfig.img) == 0 {
+			log.Error("Empty image name. Are you missing the image flag?")
+			return
+		}
+
```

Comment:
Without the above the error message was `Image tag is missing!` multiple times. This was confusing. In the above we check if the user specified `image:tag` with the `-i` or `--image` flags. It is also possible that the user uses the flag but passes an empty string.